### ISSUE TITLE
What is the complete training command of Pascal data set？

### ISSUE THREAD

**Issue by kongxm889** [xuannianz/EfficientDet#114]
error: argument dataset_type: invalid choice: '\r' (choose from 'coco', 'pascal', 'csv')
train.sh: 2: train.sh: --snapshot: not found
train.sh: 3: train.sh: --phi: not found
train.sh: 4: train.sh: --gpu: not found
train.sh: 5: train.sh: --random-transform: not found
train.sh: 6: train.sh: --compute-val-loss: not found
train.sh: 7: train.sh: --freeze-bn: not found
train.sh: 8: train.sh: --batch-size: not found
train.sh: 9: train.sh: --steps: not found
